app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -128,9 +127,6 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
     protected val browserToolbarView: BrowserToolbarView
         get() = _browserToolbarView!!
 
-    private val sessionManager: SessionManager
-        get() = requireComponents.core.sessionManager
-
```

Comment:
Just noting for future reference that this is a bit of extra cleanup and not central to the PR